### PR TITLE
Blog Post: 5th Wave of Nostr Grants

### PR DESCRIPTION
**Pull request by dergigi**
2nd draft. To be reviewed & sanity checked by @OpenSats/volunteers 

---

**Build preview:**

- [/blog/nostr-grants-july-2024](https://os-website-git-5th-wave-nostr-open-sats.vercel.app/blog/nostr-grants-july-2024)

**Social preview:**

- 39-nostr-2

### CODE REVIEW

File: data/blog/nostr-grants-july-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,222 @@
+---
+title: "Fifth Wave of Nostr Grants"
+date: '2024-07-11'
+tags: ['OpenSats', 'grants', 'nostr']
+draft: false
+authors: ['arvin', 'btcschellingpt']
```
Comment (Arvin21M) [inline data/blog/nostr-grants-july-2024.mdx:6]:
```suggestion
authors: ['arvin', 'jason']
```

**Review by Arvin21M — CHANGES_REQUESTED** [review]
btcschellingpt >> jason for this one 🙏 
Reference:
https://github.com/OpenSats/volunteers/issues/34#issuecomment-2191250757

**Review by Arvin21M — APPROVED** [review]
- Checked all links, they work and point to the right destination. 
- You clearly improved the intro quite a bit, so thank you for helping Jason and I look better 😄 
- Table of content links move page down to proper placement according to the headers.
- Image looks great, though I am really curious about that nostr code in the background of the image.
- Good to go! 🚀

**Comment by dergigi** [comment]
It's the `event` description of [NIP-01](https://github.com/nostr-protocol/nips/blob/master/01.md):


```jsonc
{
  "id": <32-bytes lowercase hex-encoded sha256 of the serialized event data>,
  "pubkey": <32-bytes lowercase hex-encoded public key of the event creator>,
  "created_at": <unix timestamp in seconds>,
  "kind": <integer between 0 and 65535>,
  "tags": [
    [<arbitrary string>...],
    // ...
  ],
  "content": <arbitrary string>,
  "sig": <64-bytes lowercase hex of the signature of the sha256 hash of the serialized event data, which is the same as the "id" field>
}
```

;)

**Comment by dergigi** [comment]
Live!

- https://opensats.org/blog/nostr-grants-july-2024
- TBD

**Pull request merged**